cmd/go-filecoin/config_integration_test.go

```diff
@@ -74,7 +74,7 @@ func TestConfigDaemon(t *testing.T) {
 		jsonOut := op1.ReadStdout()
 		bootstrapConfig := config.NewDefaultConfig().Bootstrap
 		bootstrapConfig.Addresses = []string{"fake1", "fake2"}
-		someJSON, err := json.MarshalIndent(bootstrapConfig, "", "\t")
+		someJSON, err := json.Marshal(bootstrapConfig)
```

Comment:
Nit: I would actually prefer that pretty JSON is the default, with a flag for compressed JSON. Can we acheive that easily?

Reply:
Yes. Pretty easily. I added a new 'pretty-json' command encoder and made 'pretty-json' the default for the `--enc` option. `--enc=json` still works and outputs condensed json.